Resources/Textures/_Harmony/Clothing/Hands/Gloves/Color/JesterPOMNI.rsi/meta.json

```
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-NC-SA-4.0",
+  "copyright": "created by SnoWDooM",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "icon"
+    },
+    {
+      "name": "equipped-HAND",
+      "directions": 4
+    },
+    {
+      "name": "inhand-left",
+      "directions": 4
+    },
+    {
+      "name": "inhand-right",
+      "directions": 4
+    }
+  ]
+}
```

```suggestion
{
    "version": 1,
    "license": "CC-BY-NC-SA-4.0",
    "copyright": "created by SnoWDooM",
    "size": {
      "x": 32,
      "y": 32
    },
    "states": [
      {
        "name": "icon"
      },
      {
        "name": "equipped-HAND",
        "directions": 4
      },
      {
        "name": "inhand-left",
        "directions": 4
      },
      {
        "name": "inhand-right",
        "directions": 4
      }
  ]
}
```
thats essentially what you need to do to every RSI file